src/app/client-view/training-view/training-day/exercise/exercise.component.ts

```diff
@@ -21,7 +21,7 @@ export class ExerciseComponent implements OnInit {
 
   max: number;
 
-  constructor(private trainingService: TrainingService, private rmService: RMCalculatorService) {}
+  constructor(private trainingService: TrainingService) {}
```

Comment:
We need this injection back! haha
```suggestion
  constructor(private trainingService: TrainingService, private rmCalcService: RMCalcService) {}
```